guardian/support-frontend
# Return user type from create subscription endpoint

## What are you doing in this PR?

Return the identity user type from the create endpoint and use this information to pass to the generic thank you page.






## How to test

Go through the generic checkout signed out and see the following messaging on the thank you page

- Has a previous guest account : “Continue to your account”
- Has an existing account: “Continue to your account”
- Has no account: “Complete your Guardian account”




## Screenshots
![image](https://github.com/user-attachments/assets/f87fa9e7-6931-4dcc-b577-f19df9473cf2)

## Next steps
Do the same for one time
